aplus/settings.py

```diff
@@ -37,6 +37,7 @@
 SHIBBOLETH_BUTTON_TEXT = 'Aalto login'
 MOOC_TITLE_TEXT = 'Users external to Aalto'
 MOOC_BODY_TEXT = 'Some of our courses are open for everyone. Log in with your user account from one of the following services.'
+MOOC_ACTION_TEXT = 'Users external to Aalto Login'
```

Comment:
https://www.w3.org/TR/wai-aria-practices/examples/landmarks/region.html. In this article they mention the use of the `aria-label`. We could use the `aria-labelledby` but in this case, the headers of each section do not mention that this is a login section. It only mentions the type of user, i.e. `'Users external to Aalto'`

Reply:
I can also hardcode the text in the `aria-label`  attribute. @Mankro which solution do you think is better?

Reply:
All right. I would imagine that when the page starts with the main title "Log in", then people understand that the section title "Aalto University users" means the login for those people without repeating the word "login". Therefore, we can use `aria-labelledby="id of the visible section heading"` in each login type section and we don't need to add new strings. I trust that this is clear because people must notice the main title "Log in" first. I can't test this with screen readers or other accessible technology, but I don't think they would just skip over the main title above the login sections. Even the metadata `<title>` html element contains the word "login" (could be changed to the verb "log in" as many other web sites seem to do so in order to emphasize the action).